__live-tests__/preference-api.test.ts

```
@@ -0,0 +1,36 @@
+// Imports
+import { AuthProvider, Environment, ApiClient, AuthScopes } from '../src/app';
+import { describe, expect, test } from 'bun:test';
+
+// Describe the Preference API
+describe('Live API: Preferences', async () => {
+	// Initialize the Auth Provider
+	const authProvider = new AuthProvider(process.env.QB_CLIENT_ID!, process.env.QB_CLIENT_SECRET!, process.env.REDIRECT_URI!, [
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)